epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/MG5aMC_patches/PROD/patch.P1

```diff
@@ -299,15 +301,7 @@ index e73e654d4..27fbe7302 100644
  C     
  C     This is just to temporarily store the reference grid for
  C      helicity of the DiscreteSampler so as to obtain its number of
-@@ -140,6 +143,7 @@ C     ----------
- C     BEGIN CODE
- C     ----------
- 
-+      call counters_smatrix1_start()
-       NTRY(IMIRROR)=NTRY(IMIRROR)+1
-       THIS_NTRY(IMIRROR) = THIS_NTRY(IMIRROR)+1
-       DO I=1,NEXTERNAL
-@@ -217,6 +221,17 @@ C     ----------
+@@ -217,6 +220,17 @@ C     ----------
```

Comment:
This is weird no?

Those lines are also heavily changed in the goodhel PR.

Reply:
Ah this is in the patch ... not in the matrix.f (not sure why I was confused)...
So this makes more sense ...

Reply:
Here I just removed the addition of a ` call counters_smatrix1_start()` line. This is a counter that is no longer used (I think StephanH had removed it from cudacpp, but it was still there in fortran). So my PR on this is actually making the patch smaller.

The other counter calls that are still there are in a template of cudacpp rather than a patch I think, but I should check.